rpcs3/Emu/RSX/Overlays/overlays.cpp

```diff
@@ -6,12 +6,12 @@ namespace rsx
 {
 	namespace overlays
 	{
-		//Singleton instance declaration
+		//	Singleton instance declaration
```

Comment:
This is just crazy looking. You can disable clang-format unless you use manual formatting to fix this kind of thing.